src/admin-views/seating/components/layout-card.php

```diff
@@ -4,5 +4,38 @@
  *
  * @since TBD
  *
- * @var Layout_Card $card The Seat Layout card to render.
+ * @var Object $card The Seat Layout card to render.
  */
+
+?>
+
+<div class="tec-tickets__tab__cards__item">
+	<div class="tec-tickets__tab__cards__item-thumbnail">
+		<img src="#" alt="<?php echo esc_attr( $card->name ); ?>">
```

Comment:
This should not be a generic object, but a dedicated value object class to:
1. Avoid having to run existence and type checks on the properties (e.g. does `name` exist? Is it a string?)
2. Avoid using public properties, this should be `$card->name()`